source/include/ota.h

```diff
@@ -353,6 +435,19 @@ OtaState_t OTA_Shutdown( uint32_t ticksToWait,
  * @brief Get the current state of the OTA agent.
  *
  * @return The current state of the OTA agent.
+ *
+ * <b>Example</b>
+ * Sleep while the OTA agent is in suspended state
+ * @code{c}
+ * // OTA Agent state
+ * OtaState_t state = OTA_GetState();
+ *
+ * while( state != OtaAgentStateSuspended )
+ * {
+ *     portSleep( SLEEP_TIME_MS );
```

Comment:
Instead of giving a specific example as it might confuse in this case that the task must sleep , have something generic //do something till agent is back in a desired state

Reply:
Done